package.json

```diff
@@ -95,6 +95,7 @@
     "node": ">= 16.0.0"
   },
   "dependencies": {
-    "@changesets/cli": "^2.26.0"
+    "@changesets/cli": "^2.26.0",
+    "next": "^13.3.0"
```

Comment:
Why is Next.js needed as a top-level dependency?

Reply:
It was one of the attempt while Vercel deployments were failing. Removed as well.